buf-generate/action.yaml

```diff
@@ -14,7 +14,7 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: "bufbuild/buf-setup-action@v1"
```

Comment:
FYI: deprecated https://github.com/bufbuild/buf-setup-action?tab=readme-ov-file#buf-setup-action

Reply:
Also, I can't find any usages of this action, but it seems like we need `setup_only: true`